apps/system/test/unit/notification_screen_test.js

```diff
@@ -31,7 +33,7 @@ var mocksForNotificationScreen = new MocksHelper([
   'SettingsListener',
   'SettingsURL',
   'Service',
-  'LazyLoader'
+  'LazyLoader',
```

Comment:
I don't know how to do this. I tried adding "mozIntl" to the list but it didn't attach it on the `window`.